tools/debugging/extract_sp_exceptions.sh

```diff
@@ -36,6 +37,9 @@ else
     DESTINATION_DIR=$(pwd)
 fi
 
+DESTINATION_DIR="$(realpath -s ${DESTINATION_DIR})/$(date +%m-%d-%Y)"
+mkdir -p $DESTINATION_DIR
+
 print_bold "Downloading PFS logs"
 
 ssh root@services-dev.raiden.network 'cd raiden-services/deployment/; docker-compose logs pfs-goerli | gzip' > ${DESTINATION_DIR}/pfs.log.gz
```

Comment:
Can you download `pfs-goerli-with-fee` here as well

Reply:
So download both?

Reply:
I'd say so, I think we use both in the scenarios

Reply:
done